src/views/new.ecr

```diff
@@ -11,6 +11,13 @@
 
       async function encryptAndStore() {
         const { key, ciphertext } = await encryptLocally(plaintext());
+        const isInputTooLong = to_base64(ciphertext).length > <%= max_ciphertext_length %>;
+        if (isInputTooLong) {
+          displayError('Your input is too long!');
+          hideSharable();
+          return;
+        }
+        hideError();
         const { url } = await storeRemotely(ciphertext, expiresInSeconds());
```

Comment:
I'd rather expect an error to be shown as aftermath caused by this `fetch` call.